pkg/service/backup/restore_worker_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2017 ScyllaDB
+
+package backup
+
+import "testing"
+
+func TestSStableID(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		in       string
+		expected string
+	}{
+		{
+			in:       "me-3g7k_098r_4wtqo2asamoc1i8h9n-big-CRC.db",
+			expected: "3g7k_098r_4wtqo2asamoc1i8h9n",
+		},
+		{
+			in:       "md-7-big-Statistics.db",
+			expected: "7",
+		},
+		{
+			in:       "mc-7-big-Statistics.db",
+			expected: "7",
+		},
+		{
+			in:       "me-7-big-TOC.txt",
+			expected: "7",
+		},
+		{
+			in:       "la-7-big-TOC.txt",
+			expected: "7",
+		},
+		{
+			in:       "keyspace1-standard1-ka-1-CRC.db",
+			expected: "1",
+		},
+	}
+
+	for id := range testCases {
+		tcase := testCases[id]
```

Comment:
```suggestion
	for _, tcase := range testCases {
```

Reply:
it is run in parallel, since tcase is a loop variable it is going to belong to upper context and be the same for every instance of the test.